claim_contracts/src/ClaimableAirdrop.sol

```diff
@@ -42,34 +55,35 @@ contract ClaimableAirdrop is
     /// @dev This initializer should be called only once.
     /// @param _owner address of the owner of the token.
     /// @param _tokenProxy address of the token contract.
-    /// @param _claimSupplier address of the wallet that has the tokens to distribute to the claimants.
+    /// @param _tokenDistributor address of the wallet that has the tokens to distribute to the claimants.
     /// @param _limitTimestampToClaim timestamp until which the claimants can claim the tokens.
     /// @param _claimMerkleRoot Merkle root of the claimants.
     function initialize(
         address _owner,
         address _tokenProxy,
-        address _claimSupplier,
+        address _tokenDistributor,
         uint256 _limitTimestampToClaim,
         bytes32 _claimMerkleRoot
-    ) public initializer {
+    ) external initializer {
         require(_owner != address(0), "Invalid owner address");
         require(
             _tokenProxy != address(0) && _tokenProxy != address(this),
             "Invalid token contract address"
         );
         require(
-            _claimSupplier != address(0) && _claimSupplier != address(this),
+            _tokenDistributor != address(0) &&
```

Comment:
Should we remove these checks? At least the `address(0)` one is included in `__Ownable_init`

Reply:
It is not strictly necessary